app/models/concerns/itemizable.rb

```diff
@@ -29,6 +29,15 @@ def quantities_by_category
         includes(:item).group("items.category").sum(:quantity)
       end
 
+      def quantities_by_name(omit_zero = false)
+        results = {}
+        each do |li|
+          next if li.quantity.zero?
```

Comment:
It looks like the `omit_zero` parameter is unused. Should that be used on this line to control if the line item should be skipped?
```
next if omit_zero && li.quantity.zero?
```